test/models/chan.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+
+var expect = require("chai").expect;
+
+var Chan = require("../../src/models/chan");
+
+describe("Chan", function() {
+	describe("#sortUsers()", function() {
+	// TODO Use this version instead when switching to irc-framework
```

Comment:
Remove these TODOs, it's unnecessary. After this PR is merged, just update the tests in ir-framework branch.